### PR TITLE
fix for svg not rendered properly, when mask is present #1177

### PR DESCRIPTION
**Pull request by AlkaBeniwal**

### CODE REVIEW

**Comment by Llorx** [comment]
Why this has not been merged? Seems like a performance improvement as no intermediate canvas is created.

**Review by Llorx — CHANGES_REQUESTED** [review]
Remove the comment as is making Trvis return an error.

File: src/Document/MaskElement.ts [hunk 1]
```diff
@@ -62,17 +62,12 @@ export default class MaskElement extends Element {
 			}) as any
 		).apply(maskCtx, 0, 0, x + width, y + height);
 
-		const tmpCanvas = document.createCanvas(x + width, y + height);
-		const tmpCtx = tmpCanvas.getContext('2d');
+        //canvas is getting blurred when mask is present in svg,using target scaled up canvas instead of temporary canvas
```
Comment (Llorx) [inline src/Document/MaskElement.ts:65]: Remove this

**Comment by dangreen** [comment]
@Llorx Hi.

> Why this has not been merged? 

Because this solution fix one case and breaks all others.

**Comment by Llorx** [comment]
Oh I see @dangreen

**Comment by AlkaBeniwal** [comment]
> @Llorx Hi.
> 
> > Why this has not been merged?
> 
> Because this solution fix one case and breaks all others.

Hi @dangreen, just wanted to know in what all cases it is breaking so that I can also try out the new fix.

File: src/Document/MaskElement.ts (hunk 1 above)
Comment (AlkaBeniwal) [reply src/Document/MaskElement.ts:65]: sure, thanks for reviewing it.

**Comment by gabelerner** [comment]
@AlkaBeniwal @Llorx closing out of staleness, please reopen against latest master and we'll see if tests pass there

**Pull request closed without merging**